docs/installation.adoc

```diff
@@ -63,7 +63,7 @@ $ brew install kadel/odo/odo
 ==== Tarball installation
 
 ----
-# sh -c 'curl -L https://github.com/openshift/odo/releases/latest/download/odo-darwin-amd64.gz | gzip -d > /usr/local/bin/odo; chmod +x /usr/local/bin/odo'
+# sh -c 'curl -L https://github.com/openshift/odo/releases/latest/download/odo-darwin-amd64.tar.gz | tar -xvzf odo-darwin-amd64.tar.gz -C /usr/local/bin; chmod +x /usr/local/bin/odo'
```

Comment:
This command is not going to work :-(

you can't use `-f` when you want to use `stdin` as an input
`-C` flag also doesn't use  like this

Reply:
ping @girishramnani

Reply:
Will take a look

Reply:
I did test them not sure why it doesn't work for you

Reply:
Will take a look again